spec/controllers/orders_controller_spec.rb

```diff
@@ -225,7 +254,7 @@
         old_card_short = test_order.buyer_card_short
         patch :update, invalid_checkout_buyer_params
 
-        expect(flash[:errors]).to include(:buyer_card_short)
+        expect(flash[:errors].keys).to include(:buyer_card_short)
```

Comment:
It seemed to be. I only changed that because I got an error message about include not being a method for flash[:errors].